apps/frontend/src/libs/hooks/use-intersection-observer/use-intersection-observer.hook.ts

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useRef } from "~/libs/hooks/hooks.js";
+
+const FIRST_ITEM = 0;
+
+type Properties = {
+	isEnabled?: boolean;
```

Comment:
let's switch it to isDisabled, to make it more clear
```suggestion
	isDisabled?: boolean;
```